pkgs/by-name/co/code-cursor/package.nix

```diff
@@ -69,12 +69,12 @@ stdenvNoCC.mkDerivation {
     ${lib.optionalString hostPlatform.isLinux ''
       cp -r bin $out/bin
       mkdir -p $out/share/cursor
-      cp -a ${appimageContents}/locales $out/share/cursor
```

Comment:
These moved to a new place in the most recent builds. Script needs to be fixed instead of just deleting stuff.